data.sql

```diff
@@ -11,3 +11,23 @@ VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (5, 'Charmander', '2020-02-08', 0, false, 11);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (6, 'Plantmon', '2021-11-15', 2, true, 5.7);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (7, 'Squirtle', '1993-05-02', 3, false, 12.13);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (8, 'Angemon', '2005-06-12', 1, true, 45);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (9, 'Boarmon', '2005-06-07', 7, true, 20.4);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (10, 'Blossom', '1998-10-13', 3, true, 17);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (11, 'Ditto', '2022-05-15', 4, true, 22);
```

Comment:
- [ ] Kindly make sure to update the day of the birthdate for "Ditto" as well it should be "2022-05-14"